### PR TITLE
[14.0][MIG] l10n_be_mis_reports_xml: migration to 14.0

### PR DESCRIPTION
**Pull request by huguesdk**

### CODE REVIEW

**Comment by huguesdk** [comment]
please note that tests are currently failing because the `mis_report_vat` report has not yet been ported to 14.0. it is being worked on in [this branch](https://github.com/coopiteasy/l10n-belgium/tree/14.0-mis_reports-add_vat_report).

**Review by robinkeunen — APPROVED** [review]
Looks good so far.

**Comment by huguesdk** [comment]
the pull request for the `mis_report_vat` report (on which this pull request depends) is ready for review: #162.

**Comment by carmenbianca** [comment]
@huguesdk #162 was merged. We should see if we can run the tests on this.

CIE T9801 suggests that there's a bug in this PR. I'm working on that now.

File: l10n_be_mis_reports_xml/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -1,5 +1,6 @@
-This modules provides xml exports the Belgium VAT declaration according
-to the official models.
+This module allows to export Belgian VAT declaration as an XML file that
+follows the official definition and can be submitted to the administration's
+online services.
 
-The xml domain definitions (xsd files) can be found on
+The XML domain definitions (XSD files) can be found on
```
Comment (victor-champonnois) [inline l10n_be_mis_reports_xml/readme/DESCRIPTION.rst:5]: Oh my god, caps ! Hugues what happened ? ^^

**Review by victor-champonnois — APPROVED** [review]

**Comment by carmenbianca** [comment]
@robinkeunen Can you please merge this?

**Pull request merged**